File: Codigo1_Java/src/No.java
public class No {
    private int numero; // Armazena o número do nó

    // Método para obter o número armazenado no nó
    public int getDado() {
        return numero;
    }

    private No aponta; // Referência para o próximo nó na lista

    // Método para obter o próximo nó na lista
    public No getProximo() {
        return aponta;
    }

    // Método para definir o próximo nó na lista
    public void setProximo(No proximo) {
        this.aponta = proximo;
    }

    // Construtor da classe No, inicializa o número do nó e define o próximo nó como
    // nulo
    public No(int numero) {
        this.numero = numero;
        this.aponta = null;
    }
}

File: Codigo2_Python/No.py
class No:
    def __init__(self, numero):
        # Método construtor que inicializa um nó com um número e um ponteiro para o próximo nó
        self.numero = numero  # Armazena o número do nó
        self.ponteiro = None  # Inicializa o ponteiro como None

    def set_ponteiro(self, ponteiro):
        # Método para definir o ponteiro para o próximo nó
        self.ponteiro = ponteiro

    def get_ponteiro(self):
        # Método para obter o ponteiro para o próximo nó
        return self.ponteiro

    def get_numero(self):
        # Método para obter o número armazenado no nó
        return self.numero
